### ISSUE TITLE
V2 Image Duplication

### ISSUE THREAD

**Issue by mlh758** [360EntSecGroup-Skylar/excelize#366]
**Description**

I initially mentioned this in #360 but this needed to be broken out into a separate issue. If you save an image in `n` sheets, each sheet will have `n` copies of that image in the cell.

**Steps to reproduce the issue:**
1. Add an image to multiple sheets and save it
2. Try moving the image in the saved document

**Describe the results you received:**

The image is duplicated (stacked on top of itself in place)

**Describe the results you expected:**

The image should be there only once

**Output of `go version`:**

```text
go version go1.12 darwin/amd64
```

**Excelize version or commit ID:**

```text
b2c12d784e94bfb14da800962c769f5a0f6f783e
```

Here is a gist with a small program to reproduce this: https://gist.github.com/mlh758/1141d5fd521b4e2aed23c7e920df5ac8

You'll need a logo.png and have to update the replace directive in `go.mod` to wherever you have this library saved on your system.

**Comment by mlh758** [comment]
Thanks for the quick fix. I pulled this down and verified it fixed the issue.
